src/vector_v4.hpp

```diff
@@ -109,6 +109,7 @@ template <typename T> class VectorV4 {
 
     // size_
     int size() { return size_; }
+
     int capacity;
     T*  temp;
```

Comment:
Why is this called temp ? Is this storing some kind of temporary data ?  I think It contains real data.